filebeat/input/filestream/fswatch.go

```diff
@@ -133,7 +134,7 @@ func (w *fileWatcher) Run(ctx unison.Canceler) {
 }
 
 func (w *fileWatcher) watch(ctx unison.Canceler) {
-	w.log.Info("Start next scan")
+    w.log.Debug("Start next scan")
```

Comment:
@kvch are you fine with making this a debug log instead of info?

Reply:
Yes, thanks.